Cargo.toml

```diff
@@ -42,6 +42,7 @@ default-members = [
 
 [workspace.lints.clippy]
 manual_c_str_literals = "allow"
+ref_as_ptr = "warn"
```

Comment:
What's this do?

Reply:
Upstream docs: [`clippy::ref_as_ptr`](https://rust-lang.github.io/rust-clippy/stable/index.html#/ref_as_ptr)

As context that I realize was missing from the OP: This is follow-up from <https://github.com/gfx-rs/wgpu/pull/6214>.

Reply:
Aha!